modules/core/src/main/java/org/apache/ignite/internal/processors/cache/verify/IdleVerifyUtility.java

```diff
@@ -73,6 +74,24 @@ public static void checkPartitionsPageCrcSum(
         IgniteThrowableSupplier<FilePageStore> pageStoreSup,
         int partId,
         byte pageType
+    ) {
+        checkPartitionsPageCrcSum(pageStoreSup, partId, pageType, null);
+    }
+
+    /**
+     * Checks CRC sum of pages with {@code pageType} page type stored in partition with {@code partId} id
+     * and associated with cache group.
+     *
+     * @param pageStoreSup Page store supplier.
+     * @param partId Partition id.
+     * @param pageType Page type. Possible types {@link PageIdAllocator#FLAG_DATA}, {@link PageIdAllocator#FLAG_IDX}
+     *      and {@link PageIdAllocator#FLAG_AUX}.
+     */
+    public static void checkPartitionsPageCrcSum(
+        IgniteThrowableSupplier<FilePageStore> pageStoreSup,
+        int partId,
+        byte pageType,
+        @Nullable IgniteThrowableBiConsumer<Integer, ByteBuffer> pagePostProcessor
```

Comment:
Why is java's `BiConsumer` not enough?

Reply:
I don't what to limit user with necessity to catch all checked exception.